Cargo.toml

```diff
@@ -2,5 +2,6 @@
 
 members = [
     "crates/core_simd",
+    "crates/std_float",
```

Comment:
Should this crate maybe be called `std_simd`?  It only has the extra float stuff for now, but it could end up with something unexpected.

Reply:
It's not necessarily obvious we would want to keep anything else we include in `std` in the same crate and we'd likely have to adjust the paths for inclusion anyways (to use the more complicated scheme). It's fine to rename it if something does come up.